main.py

```diff
@@ -1,12 +1,182 @@
 import requests
+from models.video import Video
+from models.customer import Customer
+from models.rental import Rental
 
 URL = "http://127.0.0.1:5000"
 BACKUP_URL = "https://retro-video-store-api.herokuapp.com"
 
 def main():
     print("WELCOME TO RETRO VIDEO STORE")
-    pass
+if __name__ == "__main__":
+    main()
+
+def print_stars():
+    print("\n**************************\n")
+
+def list_options():
+    options = {
+        "1": "List all videos", 
+        "2": "Create a video",
+        "3": "Select a video", 
+        "4": "Update selected video", 
+        "5": "Delete a video", 
+        "6": "List all customers",
+        "7": "Create a customer",
+        "8": "Select a customer",
+        "9": "Update a customer",
+        "10": "Delete a customer",
+        "11": "Check out a video to a customer",
+        "12": "Check in a video to a customer",
+        "13": "Exit"
+        }
+    print("What would you like to do?")
+    print_stars()
+    print("These are the actions you can perform")
+    print_stars()
+
+    for choice_num in options:
+        print(f"Option {choice_num}. {options[choice_num]}")
+    print("You must select a video or customer before altering it.")
+    print_stars()
+
+    return options
+
+def make_choice(options, video, customer): 
+    valid_choices = options.keys()
+    choice = None
+
+    while choice not in valid_choices:
+        print("What would you like to do?")
+        choice = input("Make your selection using the option number: ")
+
+    if choice in ['4','5'] and video.selected_video == None:
+        print("You must select a video before altering or deleting it.")
+        choice = "3"
+    if choice in ['9','10'] and customer.selected_customer == None:
+        print("You must select a customer before altering or deleting it.")
+        choice = "8"
+    
+    return choice
+
+def run_cli():
+    video = Video(BACKUP_URL)
+    customer = Customer(BACKUP_URL)
+    rental = Rental(BACKUP_URL)
+    print(f"this is the video: {video}")
+    options = list_options()
+
+    play = True
+    while play==True:
+        choice = make_choice(options, video, customer)
+        
+        if choice=='1':
+            print_stars()
+            print("Videos:")
+            for vid in video.list_videos():
+                print(vid)
+        elif choice=='2':
+            print("Great! Let's create a new video.")
+            title=input("What is the title of your video? ")
+            total_inventory=input("What is the total inventory of your video? ")
+            video.create_video(title=title, total_inventory=total_inventory)
+            print("New video:", title) 
+        elif choice=='3':
+            id = input("What is the id of the video you like to select? ")
+            if id.isnumeric():
+                id = int(id)
+                video.selected_video = video.get_one_video(id=id) 
+            if video.selected_video:
+                print("Selected video: ", video.selected_video)
```

Comment:
I think we need an else statement here. What happens if the user doesn't enter any data into the `input`? It will error out when `get_one_video` is executed